ddl/util/syncer_test.go

```diff
@@ -221,14 +221,14 @@ func TestSyncerSimple(t *testing.T) {
 	}
 	checkRespKV(t, 0, ttlKey, "", resp.Kvs...)
 
-	// for RemoveSelfVersionPath
+	// for Close
```

Comment:
Does it mean "test for SchemaSyncer.Close()"?

Reply:
Yes. This test is `TestSyncerSimple`, so I don't add `SchemaSyncer`.